ergo/distributions/constants.py

```diff
@@ -1,2 +1,9 @@
-# The default number of points HistogramDists use to represent a distribution
-histogram_default_num_points = 201
+# The default number of points PointDensity uses to represent a distribution
+import jax.numpy as np
+
+point_density_default_num_points = 200
+bin_sizes = np.full(
+    point_density_default_num_points, 1 / point_density_default_num_points
+)
+grid = np.linspace(0, 1, point_density_default_num_points + 1)
+target_xs = (grid[1:] + grid[:-1]) / 2
```

Comment:
Given that we're doing computations here I don't think this should go in `constants.py`.

I was hoping that the only way `point_density_default_num_points` shows up is as the default value for a keyword argument to one or two methods in `PointDensity`. Are `bin_sizes`, `grid`, and `target_xs` global for efficiency reasons? (If so, do you know it makes a difference?)

Reply:
It was for efficiency reasons. I'll profile it and see if it does actually matter significantly. If it does, I thought that, given we just want to compute it once and it is constant for all point_density distributions, constants.py was a reasonable place. Where would you prefer it?